server/ot_rpc.go

```diff
@@ -197,8 +210,15 @@ func runTranscode(n *core.LivepeerNode, orchAddr string, httpc *http.Client, not
 	if err != nil {
 		glog.Error("Error submitting results ", err)
 	} else {
-		ioutil.ReadAll(resp.Body)
+		rbody, rerr := ioutil.ReadAll(resp.Body)
 		resp.Body.Close()
+		if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+			if rerr != nil {
+				glog.Errorf("Orchestrator returned HTTP %v", resp.StatusCode)
```

Comment:
Include `rerr` in the logged message since it is non-nil here? Seems like we can do the error handling here similar to how it is done [here](https://github.com/livepeer/go-livepeer/blob/fce8a401fbd3cc24dd6341a95921bd5e63b31001/server/broadcast.go#L674) for the detection webhook response.